open-media-match/.devcontainer/devcontainer.json

```diff
@@ -31,8 +31,9 @@
     }
   },
   "remoteUser": "vscode",
-  "postCreateCommand": "pip install --editable .[dev]",
+  "postCreateCommand": "pip install --editable .",
```

Comment:
It can be super helpful to have an alias for the devtools version that installs more things than we would include in a package.

Another option is a .[all] alias.

Reply:
So I went with a `pyproject.toml` instead of a `setup.py` (which is deprecated) and this handles devtools differently. Right now the `[dev]` bit here doesn't refer to anything.